service-deployment/bootstrap/rest-server/rest-server.yaml.template

```diff
@@ -53,7 +53,7 @@ spec:
         - name: LOWDB_ADMIN
           value: {{ clusterinfo['restserverinfo']['lowdb_admin'] }}
```

Comment:
the same for username?

Reply:
If we assume some others will create a user name with digits, the answer is yes

Reply:
Updated